search/search_indexes.py

```diff
@@ -30,6 +31,12 @@ def index_queryset(self, using=None):
         """Records to check when updating entire index."""
         return self.get_model().objects.all()
 
+    def prepare_text(self, obj):  # pylint: disable=no-self-use
```

Comment:
@Ferdi Yes, it's directly related. Putting two values in the "prepare" method and creating a template are two ways to do exactly the same thing. In fact, in this PR I've removed the template that was doing exactly that in order to keep all the logic in one place.